packages/@okta/vuepress-site/docs/concepts/third-party-cookies/index.md

```diff
@@ -0,0 +1,155 @@
+
+The result is that Okta returns 403 Forbidden errors to the user or that your application repeatedly directs users back to the sign-in page.
+
+Specifically, this problem affects [certain methods](https://github.com/okta/okta-auth-js#third-party-cookies) of the Okta Auth JavaScript SDK. The [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget#okta-sign-in-widget) uses the affected Okta Auth JavaScript SDK methods internally. Any customer-developed code that directly makes XHR calls to the Okta [Sessions API](/docs/reference/api/sessions/) is also affected. 
+
+### Token renewal in customer-built SPAs that use the OAuth 2.0 implicit flow or PKCE flow
+
+If your code uses the OAuth 2.0 implicit flow or PKCE flow to handle token renewal (which typically occurs in the context of a SPA that uses the [implicit flow](/docs/guides/implement-implicit/)) or [PKCE flow](/docs/guides/implement-auth-code-pkce/), the user's browser can block Okta session cookies from being sent, so that token renewal is never successfully completed.
+
+The result is that ID tokens and access tokens expire without being renewed, and users are prompted to sign in more frequently, with the frequency determined by the token expiry time.
+
+## Workaround: Custom URL Domain
+
+You can implement a workaround by using the [custom URL domain](/docs/guides/custom-url-domain/) feature.
+
+By making your Okta org effectively part of the same domain as your application server from a browser’s perspective, use of custom URL domain moves Okta session cookies to a first-party context. Calls to Okta become calls within the same site, and browser third-party cookie blocking is no longer triggered.
+
+For example, if your original Okta org is `companyname.okta.com`, and your app server is `app.companyname.com`, you would use the custom URL domain feature to give your Okta org a new URL like `login.companyname.com`. This puts your app and your Okta org within the same site.
+
+It's okay for the Okta org and the app server to have different subdomains (in the above example, `app` and `login`). Only the effective Top-Level Domain plus the one label immediately to its left (eTLD+1) need to be the same (in the example, `companyname.com`).
+
+After setting up custom URL domain, your need to update your configuration of the Okta Sign-In Widget and of any Okta SDKs, so that they use your new custom domain as the base URL of your org.
+
+If your code makes any XHR calls directly to Okta endpoints, you need to update the URLs of those calls.
+
+The following sections show examples of these changes. The examples assume your original Okta Org URL is `companyname.okta.com` and your custom URL domain is `login.companyname.com`.
+
+### Sign-In Widget
+
+For information on updating the configuration of the Okta Sign-In Widget, see <https://github.com/okta/okta-signin-widget#basic-config-options>
```

Comment:
nit: markdown link (here and elsewhere if it's not intended)?

Reply:
Changed to markdown link.